neurom/__init__.py

```diff
@@ -57,11 +57,14 @@
 """
 
 import logging as _logging
+from morphio import SomaType, Morphology
 
-from .core import (NeuriteType, graft_neuron, iter_neurites, iter_sections,
-                   iter_segments, Neuron)
-from .core.dataformat import COLS
+from neurom.core.dataformat import COLS
+from neurom.core.types import NeuriteType
+
+from .core import graft_neuron, iter_neurites, iter_sections, iter_segments
```

Comment:
why using the relative imports ?

Reply:
because NeuroM v1. I didn't clean the code in general as I don't like to mix multiple concerns in the single PR.